### PR TITLE
Create file 2

### PR DESCRIPTION
**Pull request by Priyankasivakumar**
created file to test comments

### CODE REVIEW

File: file 2 [hunk 1]
```diff
@@ -0,0 +1 @@
+Line added
```
Comment (Priyankasivakumar) [inline file 2:1]: Commented on edited file

**Review by PriyankaParent — APPROVED** [review]

**Review by PriyankaParent — CHANGES_REQUESTED** [review]
Request changes

**Comment by PriyankaParent** [comment]
Merged

**Comment by PriyankaParent** [comment]
Merged with comment

**Pull request closed without merging**